### DISCUSSION TITLE
This user account is blocked - yes I've sent an email to the support

### DISCUSSION THREAD

**Discussion by Ilshae** [vercel/community#4597] [Account Issue]
### Question

Hello,

I want to learn nextjs and all tutorials want to deploy that stuff on Vercel and I wanted to see what this place is about. Unfortunately I still did not receive a reply to the email I sent to the support team about my problem. It's been 2 weeks.

I can't log in using my github account, it says it's blocked.

Anything else I can do? Or is it just unavailable for an unworthy mortal like me?

**Comment by amyegan** [comment]
Sorry I missed your post in the CLI repo. I'm moving it over to our support community.

If you let me know the case number, I can check on the support ticket for you!

**Reply by Ilshae** [reply]
00164169
